k8s/prod/common/xui/webapp.yaml

```
@@ -26,6 +26,10 @@ spec:
         FEATURE_TERMS_AND_CONDITIONS_ENABLED: false
         NOW: true
         FEATURE_SECURE_COOKIE_ENABLED: false
+        HEALTH_CCD_COMPONENT_API: https://gateway.ccd.platform.hmcts.net/health
+        SERVICES_CCD_COMPONENT_API: https://gateway.ccd.platform.hmcts.net
+        SERVICES_IDAM_API_URL: https://idam-api.platform.hmcts.net
+        SERVICES_IDAM_LOGIN_URL: https://idam-web-public.platform.hmcts.net
```

https://github.com/hmcts/rpx-xui-manage-organisations/blob/develop/infrastructure/prod.tfvars#L6

```suggestion
        SERVICES_IDAM_LOGIN_URL: https://hmcts-access.service.gov.uk
```